.github/workflows/ci.yaml

```diff
@@ -10,6 +10,7 @@ name: Continuous integration
 
 env:
   CARGO_TERM_COLOR: always
+  PROPTEST_CASES: 1
```

Comment:
That seems a bit too few. How about 100?

Reply:
Ok. That seems a reasonable change to add a minimum safety net for property testing.